src/api/certificates/source/v2.birth-certificate-certified-copy.svg

```diff
@@ -378,16 +378,16 @@
       font-size="8" letter-spacing="0px">
       <!-- NOTE: Since we are using intl-lookup, we need to use informant.relation as actual value
       rather than as stringified one. Hence we opt for native lookup over $lookup  -->
-      {{#ifCond (lookup $declaration 'informant.relation') '===' 'MOTHER' }}
+      {{#ifCond (lookup $declaration 'informant.relation') '===' 'Mother' }}
         <tspan x="307" y="518.104">{{$lookup $declaration 'mother.name.fullname'}}</tspan>
       {{ else }}
-        {{#ifCond (lookup $declaration 'informant.relation') '===' 'FATHER'}}
+        {{#ifCond (lookup $declaration 'informant.relation') '===' 'Father'}}
           <tspan x="307" y="518.104">{{$lookup $declaration 'father.name.fullname'}}</tspan>
         {{ else }}
           <tspan x="307" y="518.104">{{$lookup $declaration 'informant.name.fullname'}}</tspan>
         {{/ifCond}}
       {{/ifCond}}
-      <tspan x="307" y="528.104">{{$intl 'v2.constants.informant' (lookup $declaration 'informant.relation')}}{{#ifCond (lookup $declaration 'informant.relation') '===' 'OTHER'}} ({{lookup $declaration 'informant.other.relation'}}) {{/ifCond}}</tspan>
+      <tspan x="307" y="528.104">{{lookup $declaration 'informant.relation'}}{{#ifCond (lookup $declaration 'informant.relation') '===' 'OTHER'}} ({{lookup $declaration 'informant.other.relation'}}) {{/ifCond}}</tspan>
```

Comment:
Inspect why value of `informant.relation` could be 'Mother', 'Father' and 'OTHER' as mentioned [here](https://github.com/opencrvs/opencrvs-countryconfig/pull/1052#discussion_r2391006775)